Working branch

valegale opened the pull request.

hoelzer commented:
Notes:

* make a 0.0.1 release after merging into `main`. In the README, I am also using `-r 0.0.1` as example which is better than running the pipeline from cloning the repo
* Is the conda manual installation of the dependencies part necessary? It might cause more trouble bc users might install different tool versions via conda (when not specified) and might break the pipeline. I would rather remove that part from the README and tell users they need either Docker or Singularity. Especially when we add Dorado, using containers is the best way. We can also keep the conda manual dependency part if you like but I would not recommend it (this is how I wrote it now in the current README)
* I changed the README to install the pipeline via Nextflow `pull`. Otherwise, we would need to explain to users how to clone the repo, how to use the `main.nf` for `nextflow run`, etc... also by that we have more control of pipeline versions users are using ("did you use release x.y.z?"). I still added that `git clone` is possible but not recommended
* added also `--percent_cutoff_modification_table` to the `--help`: maybe this parameter can be also better explained in the `main.nf` file in the ``--help` section

valegale commented:
I think for now it's ok to keep the conda part, with the not recommended indication. I changed a bit the description of the parameters in main and I added the license
